pyfr/solvers/baseadvecdiff/inters.py

```diff
@@ -10,6 +10,8 @@ def __init__(self, be, lhs, rhs, elemap, cfg):
         # Generate the additional view matrices
         self._vect_lhs = self._vect_view(lhs, 'get_vect_fpts_for_inter')
         self._vect_rhs = self._vect_view(rhs, 'get_vect_fpts_for_inter')
+        self._comm_lhs = self._scal_view(lhs, 'get_comm_fpts_for_inter')
```

Comment:
Can you confirm that this does not result in additional memory allocations?

Reply:
Yep, this doesn't result in any more memory as some memory (`vect_fpts`) is removed and some is added (`comm_fpts`).